### DISCUSSION TITLE
Multiple users?

### DISCUSSION THREAD

**Discussion by delphiactual** [Taxel/PlexTraktSync#601] [Q&A] [answered]
Is it possible to configure this for multiple users or just separate instances per user?
If so, how to separate instances for pipx install?

**Comment by simonc56** [comment, marked as answer]
Did you try with PTS_CONFIG_DIR : 
- https://github.com/Taxel/PlexTraktSync/issues/629#issuecomment-996133715

PR are accepted to improve readme file 😉
